src/vs/workbench/services/outline/browser/outline.ts

```diff
@@ -20,6 +20,7 @@ export const IOutlineService = createDecorator<IOutlineService>('IOutlineService
 export const enum OutlineTarget {
 	OutlinePane = 1,
 	Breadcrumbs = 2,
+	NotebookStickyScroll = 3,
```

Comment:
Can this be a concept in notebook only?

Reply:
yes-ish? Started to create a nbOutlineTarget enum and pass that around to all of the notebookOutline areas, but that change felt a bit involved for right now. This target only controls whether the outline created includes code cells, which in the stickyscroll case it does need to. 

Now I pass in the quickpick enum, as this leaves codecells as true. the other two fields defer to the settings for showCodeCells for those respective widgets, which can break sticky's functionality quite easily.

The easiest change is just keeping this `OutlineTarget.NotebookStickyScroll` field, but I understand that we want to avoid leaking the notebook concept outside of its scope. Passing quickpick is effectively the same thing, and I'll comment it for now